lib/util/DataURI.js

```diff
@@ -6,7 +6,7 @@
 
 // data URL scheme: "data:text/javascript;charset=utf-8;base64,some-string"
 // http://www.ietf.org/rfc/rfc2397.txt
-const URIRegEx = /^data:([^;,]+)?((?:;(?:[^;,]+))*?)(;base64)?,(.*)$/;
+const URIRegEx = /^data:([^;,]+)?((?:;(?:[^;,]+))*?)(;base64)?,(.*)$/i;
```

Comment:
Would you please add unit tests for this change? E.g. parsing URLs like `DATA:text/javascript;charset=UTF-8,module.exports={number:42,fn:()=>\"Hello world\"}`.

Reply:
And one more thing, would you please make this change as a separate commit? Now it can be difficult to understand how this change is connected with  absolute url specifier support.

Reply:
added

Reply:
Good job!

I think it's better to add separate tests instead of modifying existing ones: it will be easier to understand that these new tests check case insensitivity, not CoffeeScript or JSON loading. And changing regex is still in the `add absolute url specifier support` commit. I think there is no connection between absolute url specifier and modification of this regex. It will be very difficult to understand the meaning of this change after a few months.